Daf.Meta.Editor/ViewModels/MainViewModel.cs

```diff
@@ -448,15 +488,17 @@ private void CopyDataSource(Type? windowType)
 
 			Connection? connection = null;
 
-			if (SelectedDataSourceSingle is RestDataSource rest)
+			// TODO: This could be changed to a switch, but only if there's a Connection property on DataSource/DataSourceViewModel.
+			// Does every DataSource type need a connection?
```

Comment:
Should this still be fixed?

Not every data source has a connection. Files don't.

Reply:
Does not really make a difference, I think. I've removed the comment.